x-pack/legacy/plugins/apm/public/components/app/Main/route_config/route_names.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const HOME = 'home';
+export const SERVICES = 'services';
+export const TRACES = 'traces';
+export const SERVICE = 'service';
+export const TRANSACTIONS = 'transactions';
+export const ERRORS = 'errors';
+export const ERROR = 'error';
+export const METRICS = 'metrics';
+export const TRANSACTION_TYPE = 'transaction_type';
+export const TRANSACTION_NAME = 'transaction_name';
```

Comment:
Have you considered storing these values in an enum?
Example:
```typescript
export const enum RouteName {
  HOME = 'home',
  SERVICES = 'services',
  ...
}
```
Then we could defined `BreadcrumbRoute` with member `name: RouteName;` and enforce with types that only valid `RouteName`s can be used.

Reply:
Aye, much better 👍